extensions/openwsman/processors/SourceInitiatedSubscriptionListener.cpp

```diff
@@ -248,9 +248,9 @@ bool SourceInitiatedSubscriptionListener::loadState() {
   return true;
 }
 
-std::string SourceInitiatedSubscriptionListener::Handler::millisecondsToXsdDuration(int64_t milliseconds) {
+std::string SourceInitiatedSubscriptionListener::Handler::millisecondsToXsdDuration(std::chrono::milliseconds milliseconds) {
   char buf[1024];
-  snprintf(buf, sizeof(buf), "PT%" PRId64 ".%03" PRId64 "S", milliseconds / 1000, milliseconds % 1000);
+  snprintf(buf, sizeof(buf), "PT%" PRId64 ".%03" PRId64 "S", milliseconds.count() / 1000, milliseconds.count() % 1000);
```

Comment:
the requirement for `std::chrono::milliseconds::rep` is that it is a signed integer type of at least 45 bits, on most hardware this probably means `int64_t` but it might not be, a cast would help ensure that `PRId64` is the correct format specifier

Reply:
changed all printf like usages to use the `PRId64` and `static_cast<int64_t>` in https://github.com/apache/nifi-minifi-cpp/pull/1225/commits/78f8658fde802c73a5965dd734a544acbc37dca3 and https://github.com/apache/nifi-minifi-cpp/pull/1225/commits/1fcbcc05ce27c9f7386c8f5b3a4a5267593c8fa1

Reply:
`static_cast` works, but wrapping the number in a braced init style conversion (i.e. `int64_t{milliseconds.count()}`) is better IMO, because if if ever becomes wider than `int64_t`, then that will result in a compiler error. (Which is better than silently doing narrowing conversion IMO.)

Reply:
good idea, changed to the suggested format in https://github.com/apache/nifi-minifi-cpp/pull/1225/commits/078ef637a1043f6e41d9d383833620cf6202f7e5